virttest/virt_admin.py

```diff
@@ -868,5 +868,5 @@ def client_disconnect(server_name, client_id, **dargs):
     :param dargs: standardized virt-admin function API keywords.
     :return: CmdResult object
     """
-    cmd = "client_disconnect %s %s" % (server_name, client_id)
+    cmd = "client-disconnect %s %s" % (server_name, client_id)
```

Comment:
@balamuruhans Thanks very much for your patience. I have modified my docstring according your comments.